<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QWidget" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gracz</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="ans1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Answer 1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Connect to server</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="ans4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Answer 4</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="ans2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Answer 2</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="gameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Connect to game</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="chatView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="ans3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Answer 3</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
